### ISSUE TITLE
Install issue with 0.13.4-rc.0

### ISSUE THREAD

**Issue by mhuebert** [oliviertassinari/react-swipeable-views#563]
In latest release, I am seeing `Could not install from "node_modules/react-swipeable-views/packages/react-swipeable-views" as it does not contain a package.json file.` when I try to install.

(I am needing a release that includes https://github.com/oliviertassinari/react-swipeable-views/pull/556, because of version conflicts related to the dom-helpers library, which introduced unfortunate breaking changes awhile back)

- [x] I have searched the [issues](https://github.com/oliviertassinari/react-swipeable-views/issues) of this repository and believe that this is not a duplicate.

## Steps to Reproduce (for bugs)
Add `"react-swipeable-views": "0.13.4-rc.0",` to package.json and `npm install`

## Your Environment
npm 6.4.1
Mac OS X 10.5.1

**Comment by ichirkin** [comment]
Me too:
I try: 
yarn add react-swipeable-views
```
yarn add v1.21.1
[1/4] 🔍  Resolving packages...
error Package "react-swipeable-views" refers to a non-existing file '".../packages/react-swipeable-views"'.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
Error: Package "react-swipeable-views-core" refers to a non-existing file '".../packages/react-swipeable-views-core"'.
    at MessageError.ExtendableBuiltin (/usr/local/lib/node_modules/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/local/lib/node_modules/yarn/lib/cli.js:750:123)
    at FileResolver.<anonymous> (/usr/local/lib/node_modules/yarn/lib/cli.js:50229:15)
    at Generator.next (<anonymous>)
    at step (/usr/local/lib/node_modules/yarn/lib/cli.js:310:30)
    at /usr/local/lib/node_modules/yarn/lib/cli.js:321:13
Error: Package "react-swipeable-views-utils" refers to a non-existing file '".../packages/react-swipeable-views-utils"'.
    at MessageError.ExtendableBuiltin (/usr/local/lib/node_modules/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/local/lib/node_modules/yarn/lib/cli.js:750:123)
    at FileResolver.<anonymous> (/usr/local/lib/node_modules/yarn/lib/cli.js:50229:15)
    at Generator.next (<anonymous>)
    at step (/usr/local/lib/node_modules/yarn/lib/cli.js:310:30)
    at /usr/local/lib/node_modules/yarn/lib/cli.js:321:13
```

**Comment by sarangjo** [comment]
Same here, on Ubuntu 18 (WSL).
```
$ yarn add react-swipeable-views
yarn add v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
error Package "react-swipeable-views" refers to a non-existing file '"/mnt/c/Users/.../packages/react-swipeable-views"'.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
Error: Package "react-swipeable-views-core" refers to a non-existing file '"/mnt/c/Users/.../packages/react-swipeable-views-core"'.
    at MessageError.ExtendableBuiltin (/usr/share/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/share/yarn/lib/cli.js:750:123)
    at FileResolver.<anonymous> (/usr/share/yarn/lib/cli.js:50229:15)
    at Generator.next (<anonymous>)
    at step (/usr/share/yarn/lib/cli.js:310:30)
    at /usr/share/yarn/lib/cli.js:321:13
Error: Package "react-swipeable-views-utils" refers to a non-existing file '"/mnt/c/Users/.../packages/react-swipeable-views-utils"'.
    at MessageError.ExtendableBuiltin (/usr/share/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/share/yarn/lib/cli.js:750:123)
    at FileResolver.<anonymous> (/usr/share/yarn/lib/cli.js:50229:15)
    at Generator.next (<anonymous>)
    at step (/usr/share/yarn/lib/cli.js:310:30)
    at /usr/share/yarn/lib/cli.js:321:13
```

**Comment by chandankumarshanbhag** [comment]
npm install --save react-swipeable-views@0.13.3

just install the old version that works fine

**Comment by sarangjo** [comment]
I did install the old version, but it's something to be fixed for 0.13.4 and beyond. Just wanted to add my platform to the conversation as well.

**Comment by robsco-git** [comment]
Does anybody know where the source code for the 0.13.4 RC releases is? Looks like @jeanpoelie [published them](https://www.npmjs.com/~jeanpoelie) but I can't find the commits from 0.13.3 to the 0.13.4 RCs...

**Comment by jeanpoelie** [comment]
@robsco-git unfortunatly after checking this weekend i made a mistake with the release where i didnt use the proper command (first timer mistskes haha). Therefore i will need to publish another RC soon.
Unfortunatly i have had some family issues this weekend and didnt have time to resolve a new build.

I will try and do so asap but the family issues arent over yet and I have no idea when ill be near a mac pc to run the commands.

I will report back when I made a succesfull publish. 

Sorry for the delay!

**Comment by robsco-git** [comment]
Hi @jeanpoelie. Thanks for the feedback. No sweat. Good luck with the family issues.

A note for future reference; one can unpublish a NPM package within the first 72 hours: https://docs.npmjs.com/unpublishing-packages-from-the-registry

**Comment by jeanpoelie** [comment]
I will take a look at that asap, is that npm unpublish [version]? That you know off




Outlook for Android downloaden







On Mon, Jan 13, 2020 at 8:08 AM +0100, "Robert Scott" <notifications@github.com> wrote:












Does anybody know where the source code for the 0.13.4 RC releases is? Looks like @jeanpoelie published them but I can't find the commits from 0.13.3 to the 0.13.4 RCs...

**Comment by robsco-git** [comment]
> If the package is still within the first 72 hours, [...]
> npm unpublish <package_name>@<version> to remove a specific version

From https://www.npmjs.com/policies/unpublish

**Comment by jeanpoelie** [comment]
We will be releasing another version soon (this week) that addresses all issues with installing

**Comment by hcygit** [comment]
release 0.13.4 still have some issues

===
./node_modules/react-swipeable-views/lib/SwipeableViews.jsModule not found: Error: Can't resolve 'dom-helpers/events/off' in 'xxx/node_modules/react-swipeable-views/lib'./node_modules/react-swipeable-views/lib/SwipeableViews.jsModule not found: Error: Can't resolve 'dom-helpers/events/on' in 'xxx/node_modules/react-swipeable-views/lib'./node_modules/react-swipeable-views/lib/SwipeableViews.jsModule not found: Error: Can't resolve 'dom-helpers/transition/properties' in 'xxx/node_modules/react-swipeable-views/lib'
===

my environment is:
React 16.12.0
nodejs 12.14.1
macOS 10.15.2

please help to fix it

**Comment by phoqe** [comment]
#566 Created a new issue for the dom-helpers thingy

**Comment by jinsley8** [comment]
I had to revert back to release 0.13.1 which was working before I updated but I'm still getting the error.

`Module not found: Can't resolve 'dom-helpers/events/off' in './client/node_modules/react-swipeable-views/lib'`

**Comment by oliviertassinari** [comment]
Looking at the changes since 0.13.3, I could notice a couple of regressions on the demos: (tab controllable behavior, virtualization jumps, animation height no longer animating). In this context, I think that we should be less ambitious. I have reset hard master on 0.13.3 and published 0.13.7.

@jeanpoelie I have created a branch `master-off-track` with the latest changes. However, that contains the regressions too. I would suggest the following action plan:

- Something has changed with dom-helper. It's probably my fault, I shouldn't have imported deep modules: #567. The best option will be to remove 'dom-helper' as a dependency, actually, we can do the same with 'warning'.
- We revisit each change to identify where the regressions are coming from, we might have other. The simplest way would be to make small releases, one step at the time.

:)
